pkg/monitor/recorder_jsonl.go

```diff
@@ -81,6 +82,7 @@ func (m *jsonlRecorder) writeInterval(interval *monitorapi.Interval) {
 	if _, err := m.outfile.Write([]byte(fmt.Sprintf("%v\n", string(intervalJSON)))); err != nil {
```

Comment:
How many intervals are recorded? The conversion `string(intervalJSON)` allocates a new string, the call to `Sprintf` allocates a new string, and the conversion to `[]byte` allocates a new `[]byte`.

Reply:
>How many intervals are recorded? 

Disabled the cleanup to find that out.